README.md

```diff
@@ -16,61 +16,75 @@ Inspired by [Hound][].
 
 ## Installation
 
-    gem install face_control
+```bash
+gem install face_control
+```
 
-You also need to have CoffeeLint installed and available in PATH.
+You also need to have CoffeeLint installed and available in `PATH`.
 
 ## Usage
 
-    face-control <project> <repository> <pull_request_id>
+```bash
+face-control <project> <repository> <pull_request_id>
+```
+
+It's natural to run this on a continuous integration server (see “[Example](#example)” below).
 
-It's natural to run this on a continuous integration server.
-For example, here's a [Jenkins][] project setup:
+If you don't want to receive RuboCop comments with certain severity level,
+pass the severity in the `--skip-severity` option like so:
+
+```bash
+face-control --skip-severity convention <project> <repository> <pull_request_id>
+```
 
-* Source Code Management
-  * Git
-    * Repositories
-      * Refspec:
+Instead of `--skip-severity` you can use just `-S`.
+
+You can also pass multiple severity levels as a comma-separated list:
 
-              +refs/pull-requests/*:refs/remotes/origin/pull-requests/*
+```bash
+face-control -S convention,refactor <project> <repository> <pull_request_id>
+```
 
-        (make Jenkins fetch otherwise ignored Stash-created branches)
+`face-control` uses the same configuration file (`~/.stashconfig.yml`)
+as the official [Bitbucket Server Command Line Tools][]
+to connect to your Stash instance.
 
-    * Branches to build
-      * Branch Specifier:
+## Example
 
-              origin/pull-requests/*/merge
+Here's a [Jenkins][] project setup as an example:
 
-        (merge results of open non-conflicting pull requests)
+_Source Code Management → Git → Repositories → Refspec:_
```

Comment:
I've made it plain, because the highly nested list didn't look nice. But I can revert it if you think that it was better.